lib/aadutils.js

```diff
@@ -22,6 +22,7 @@
  */
 'use strict';
 
+const base64url = require('base64url');
 const crypto = require('crypto');
```

Comment:
Never mind. It is the node js inbuilt library: https://nodejs.org/api/crypto.html.